server/package.json

```diff
@@ -28,7 +28,7 @@
   },
   "scripts": {
     "prepublishOnly": "cd .. && ./scripts/build.sh",
-    "start": "node-dev server.js --dir ../db --port 3010 --debug --base-url '/sqlpad'",
+    "start": "node-dev server.js --dir ../db --port 3010 --debug --base-url /sqlpad",
```

Comment:
Notice how base-url is not quoted? On Windows/MINGW at least, these single quotes were treated as part of the URL.

Reply:
I was going to ask about this. If I recall not having the single quote might cause problems on OS X. Not a huge deal.